odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/routes/ServiceRequestRouting.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.odoo.openmrs.routes;
+
+import com.ozonehis.eip.odoo.openmrs.Constants;
+import com.ozonehis.eip.odoo.openmrs.processors.ServiceRequestProcessor;
+import lombok.Setter;
+import org.apache.camel.LoggingLevel;
+import org.apache.camel.builder.RouteBuilder;
+import org.hl7.fhir.r4.model.ServiceRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Setter
+@Component
+public class ServiceRequestRouting extends RouteBuilder {
+
+    private static final String SERVICE_REQUEST_ID = "service.request.id";
+
+    private static final String SERVICE_REQUEST_INCLUDE_PARAMS = "ServiceRequest:encounter,ServiceRequest:patient";
+
+    private static final String SEARCH_PARAMS =
+            "id=${exchangeProperty." + SERVICE_REQUEST_ID + "}&resource=${exchangeProperty."
+                    + Constants.FHIR_RESOURCE_TYPE + "}&include=" + SERVICE_REQUEST_INCLUDE_PARAMS;
+
+    @Autowired
+    private ServiceRequestProcessor serviceRequestProcessor;
+
+    @Override
+    public void configure() {
+        // spotless:off
```

Comment:
Why don't you disable this the IDE instead of the code?

Reply:
Not sure about this. I think it is something specific to Camel Routes saw a ticket regarding this but not sure.
@corneliouzbett do you know why is this required.

Reply:
We are using Spotless plugin for code formatting, which is tied to the Maven `install` phase. So, every time you run `mvn install`, it formats the code, overriding the IDE settings.The `palantir-java-format` we're using, based on the `google-java-format`, arranges the Camel routes in a straight line, which I didn't like. I'm not sure if this issue has been resolved yet. That's why we are disabling it.